Mathlib/Data/List/Intervals.lean

```diff
@@ -79,7 +79,7 @@ theorem eq_empty_iff {n m : ℕ} : Ico n m = [] ↔ m ≤ n :=
 theorem append_consecutive {n m l : ℕ} (hnm : n ≤ m) (hml : m ≤ l) :
     Ico n m ++ Ico m l = Ico n l := by
   dsimp only [Ico]
-  convert range'_append n (m-n) (l-m) 1 using 2
+  convert range'_append using 2
```

Comment:
All those arguments should stay explicit in `range'_append` since it's a rewriting lemma